transformer_engine/common/fused_attn/fused_attn.cpp

```diff
@@ -105,8 +105,8 @@ NVTE_Fused_Attn_Backend nvte_get_fused_attn_backend(
     bool flag_m512 = false;
     bool flag_arb = false;
     if ((sm_arch_ == 80 || sm_arch_ == 90)
-            && (max_seqlen_q <= 512)
-            && (max_seqlen_kv <= 512)
+            && (max_seqlen_q <= 512 && max_seqlen_q % 64 == 0)
```

Comment:
@zlsh80826 You might have already tested this locally but we should trigger the PyTorch and Paddle CIs just to have a record of clean results saved on GitHub before merging.